packages/wonder-blocks-core/src/hooks/use-is-mounted.ts

```diff
@@ -1,22 +1,28 @@
 import * as React from "react";
 
-import {useOnMountEffect} from "./use-on-mount-effect";
```

Comment:
**question:** should `useOnMounted` be updated to use `useIsomorphicLayoutEffect` internally?

Reply:
No, I don't think so. I think that it should still use `useEffect`. That's the path of least surprise and what most folks are expecting when they use it, in terms of order of execution.